.packit.yaml

```diff
@@ -17,3 +17,9 @@ jobs:
     metadata:
       targets:
         - fedora-all
+
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-eln
```

Comment:
Similar to https://github.com/rhinstaller/anaconda/pull/2868 , as the trigger is the same, why not add this to the existing `job: tests` stanza as a second target?

Reply:
Argh, I have worked with GitHub actions for too long, where the "job" name is just a human identifier. For packit it actually means something, sorry.